crossbar/common/checkconfig.py

```diff
@@ -2775,8 +2775,7 @@ def upgrade_config_file(configfile):
 
         version += 1
 
-    config[u'version'] = version
-    config.move_to_end(u'version', last=False)
+    config = OrderedDict([(u'version', version)] + [i for i in config.items()])
 
```

Comment:
Can the list-comprehension be simplified to ``list(config.items())``? Not a big deal either way though. :)